.github/workflows/ci-check-repo.yaml

```diff
@@ -0,0 +1,33 @@
+name: Check Dolt Repo
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    name: Check Repo
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Go 1.13
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.13
```

Comment:
must be `1.13` as `^1.13` resulted in a flag error for `go get -mod=readonly ./...`

Reply:
Can we upgrade the script to work with go > `1.13`? Maybe it's fine to just drop this statement in general. Or do `GOFLAGS="-mod readonly" go build ./...` to check everything builds without modifications to `go.mod`?